Code/Legacy/CrySystem/SystemWin32.cpp

```diff
@@ -171,12 +170,12 @@ int CSystem::GetApplicationInstance()
     // this code below essentially "locks" an instance of the USER folder to a specific running application
     if (m_iApplicationInstance == -1)
     {
-        string suffix;
+        AZStd::wstring suffix;
         for (int instance = 0;; ++instance)
         {
-            suffix.Format("(%d)", instance);
+            suffix = AZStd::wstring::format(L"LumberyardApplication(%d)", instance);
```

Comment:
Hmm, `"O3DEApplication"` instead?

Reply:
updated, BTW, this was just moved from line 179 before